<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_add_engine</class>
 <widget class="QDialog" name="dialog_add_engine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="acceptButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="CancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Мощность, Квт</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="Power"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="TypeEngine"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Тип двигателя</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Частота вращения, об/мин</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="FuelConsumption"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="Torque"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Удельный расход топлива, г/Квт * ч</string>
         </property>
         <property name="buddy">
          <cstring>label_3</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="MarkaEngine"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Марка двигателя</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialog_add_engine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>dialog_add_engine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
